.github/workflows/test.yml

```diff
@@ -22,5 +21,5 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: npm ci
-      - run: npm run validate
-      - run: npm run prettier
+      - run: npm run test-formatting
+      - run: npm run test-validate-rules
```

Comment:
TIL: https://docs.npmjs.com/cli/v6/commands/npm-install-ci-test

Not sure if this is _too_ magic, but <kbd>npm cit</kbd> looks neat:
> _"This command runs <kbd>npm ci</kbd> followed immediately by <kbd>npm test</kbd>."_

Where <kbd>npm test</kbd> is currently an alias for:
```js
"test": "npm run test-formatting && npm run test-validate-rules",
```

Reply:
Neat! However I prefer to keep them as separate runs in CI then it's easier to see which one failed.